src/main/java/uk/gov/hmcts/reform/cmc/model/ClaimantResponse.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.cmc.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ClaimantResponse {
+
+    private  ClaimantResponseType type;
+    private BigDecimal amountPaid;
+    private  String paymentReceived;
```

Comment:
there is extra space in here

Reply:
That is what civil service requires us as formatting. otherwise the build on formatting will fail